### PR TITLE
Fix script hashes

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: src/Home.tsx [hunk 1]
```diff
@@ -200,8 +200,8 @@ const Home = ({
 											{data.contract && data.contract[activeNet].address}
 										</Heading>
 										<Heading subtitle size={6}>
-											<span>{`Hash script: `}</span>
-											{data.contract && data.contract[activeNet].script_hash}
+											<span>{`Script hash: `}</span>
+											0x{data.contract && data.contract[activeNet].script_hash}
```
Comment (roman-khimov) [inline src/Home.tsx:204]: BTW, maybe this breaks TypeScript, I don't know.

Comment (roman-khimov) [reply src/Home.tsx:204]: The idea is to have something like https://dora.coz.io/contract/neo3/mainnet/0x2cafa46838e8b564468ebd868dcafdd99dce6221

Comment (mike-petrov) [reply src/Home.tsx:204]: Yeah, it should be
```
{`0x${data.contract && data.contract[activeNet].script_hash}`}
```

Comment (roman-khimov) [reply src/Home.tsx]: Fixed.

**Review by mike-petrov — APPROVED** [review]

**Pull request merged**